spark-indexer/src/main/java/org/uniprot/store/spark/indexer/uniprot/mapper/ChebiToUniProtDocument.java

```diff
@@ -41,12 +46,22 @@ public UniProtDocument call(Tuple2<UniProtDocument, Optional<Iterable<ChebiEntry
                 if (Utils.notNullNotEmpty(doc.cofactorChebi)) {
                     addCofactorChebi(doc, mappedChebi);
                 }
+                if (doc.commentMap.containsKey(CC_COFACTOR_CHEBI_EXP)) {
```

Comment:
these if checks can probably be replaced with a map of <String, Consumer,String>>